### PR TITLE
Revised to whet->when, what, wet,

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -13748,7 +13748,7 @@ wherease->whereas
 whereever->wherever
 wherether->whether
 whery->where
-whet->when
+whet->when, what, wet,
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `whet` is also a word

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: I'll add it a result, is that OK?

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: Sure

**Comment by luzpaz** [comment]
rebased and ready for merge

**Pull request merged**